.travis.yml

```diff
@@ -14,6 +14,9 @@ install:
   - echo 'nop'
 
 script:
+  - mkdir -p  $HOME/gopath/src/github.com/coreos/clair/
+  - rsync -az ${TRAVIS_BUILD_DIR}/ $HOME/gopath/src/github.com/coreos/clair/
+  - cd $HOME/gopath/src/github.com/coreos/clair/
```

Comment:
Would https://github.com/coreos/clair/blob/efa0d85/.travis.yml#L12 be sufficient?